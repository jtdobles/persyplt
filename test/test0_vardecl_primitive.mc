void main(){
//  freely typed, not implemented  var a = 1; 
    int b = 2; //typed
//    printf(a);
    printi(b);

//    var c = 3.0;
    float d = 4.0;
//    printf(c);
    printf(d);

//    var e = true;
    bool f = false;
 //   printf(e);
    printf(f);

//    var g = "5";
    string h = "6";
//    printf(g);
    prints(h);

}