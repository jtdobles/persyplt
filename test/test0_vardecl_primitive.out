2
4.0
false
'6'